### PR TITLE
Release/doctrine orm module 0.9.0

### PR DESCRIPTION
**Pull request by gianarb**
New release!
https://github.com/doctrine/DoctrineORMModule/releases/tag/0.9.0

### CODE REVIEW

File: site/2015/05/05/doctrine-orm-module-release-0-9-0.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+DoctrineORMModule release 0.9.0
+===============================
+
+The **Zend Framework Integration Team** is happy to announce the new release of **DoctrineORMModule**.
```
Comment (Ocramius) [inline site/2015/05/05/doctrine-orm-module-release-0-9-0.rst]: of **the** door

File: site/2015/05/05/doctrine-orm-module-release-0-9-0.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
+DoctrineORMModule release 0.9.0
+===============================
+
+The **Zend Framework Integration Team** is happy to announce the new release of **DoctrineORMModule**.  
+DoctrineORMModule 0.9.0 is out of door!!  
+
+*Note* This is the last version that supports doctrine\migrations we are working to split this feature in another indipendent module.
```
Comment (Ocramius) [inline site/2015/05/05/doctrine-orm-module-release-0-9-0.rst]: `doctrine/migrations` (forward slash). Also, add a link

Comment (Ocramius) [inline site/2015/05/05/doctrine-orm-module-release-0-9-0.rst]: Note that

Comment (Ocramius) [inline site/2015/05/05/doctrine-orm-module-release-0-9-0.rst]: `doctrine/migrations. We` (add a period)

File: site/2015/05/05/doctrine-orm-module-release-0-9-0.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
+DoctrineORMModule release 0.9.0
+===============================
+
+The **Zend Framework Integration Team** is happy to announce the new release of **DoctrineORMModule**.  
+DoctrineORMModule 0.9.0 is out of door!!  
+
+*Note* This is the last version that supports doctrine\migrations we are working to split this feature in another indipendent module.  
+Follow issue `#401 <https://github.com/doctrine/DoctrineORMModule/pull/401>`_.
+
+Following issues were solved in this release:
```
Comment (Ocramius) [inline site/2015/05/05/doctrine-orm-module-release-0-9-0.rst]: The following

File: site/2015/05/05/doctrine-orm-module-release-0-9-0.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+ - `[#380] Fix typo in configuration file markdown. <https://github.com/doctrine/DoctrineORMModule/pull/380>`_
+ - `[#385] Allow symfony 3.0 components <https://github.com/doctrine/DoctrineORMModule/pull/385>`_
+ - `[#388] update comment block in Module.php as no Module::getAutoloaderConfig()  <https://github.com/doctrine/DoctrineORMModule/pull/388>`_
+ - `[#389] Delete Module.php in root directory <https://github.com/doctrine/DoctrineORMModule/pull/389>`_
+ - `[#390] travis: PHP 5.6, 7.0 nightly added, 5.3 dropped <https://github.com/doctrine/DoctrineORMModule/pull/390>`_
+ - `[#392] Use-case: caching module' s configuration <https://github.com/doctrine/DoctrineORMModule/pull/392>`_
+ - `[#396] Removed unnecessary line in travis config <https://github.com/doctrine/DoctrineORMModule/pull/396>`_
+ - `[#398] Composer * -update for stable version <https://github.com/doctrine/DoctrineORMModule/pull/398>`_
+
+To install this version, simply update your `composer.json`:
```
Comment (Ocramius) [inline site/2015/05/05/doctrine-orm-module-release-0-9-0.rst]: .rst requires double ticks for inline code snippets

File: site/2015/05/05/doctrine-orm-module-release-0-9-0.rst (hunk 2 above)
Comment (deeky666) [inline site/2015/05/05/doctrine-orm-module-release-0-9-0.rst]: Typo: `independent`
Replace backslash: `doctrine/migrations`

Also I would reword to `we are working on extracting this feature into another`

Comment (deeky666) [inline site/2015/05/05/doctrine-orm-module-release-0-9-0.rst]: Fullstop before `we are working`

**Comment by gianarb** [comment]
Thanks and sorry for my english :)

**Comment by Ocramius** [comment]
No need to be sorry, you're doing the actual work after all! :-P

File: site/2015/05/05/doctrine-orm-module-release-0-9-0.rst [hunk 5]
```diff
@@ -0,0 +1,59 @@
+DoctrineORMModule release 0.9.0
+===============================
+
+The **Zend Framework Integration Team** is happy to announce the new release of **DoctrineORMModule**.
+
+DoctrineORMModule 0.9.0 is out of the door!!
+
+*Note* that this is the last version that supports `doctrine/migrations <https://github.com/doctrine/migrations>`_. We are working on extracting this feature into independent module.
```
Comment (deeky666) [inline site/2015/05/05/doctrine-orm-module-release-0-9-0.rst]: Last nitpick: `extracting this feature into an independent module` (missed the `an`) ;)

**Comment by deeky666** [comment]
@gianarb no worries, my english isn't better either :P Besides my last nitpick: :+1:

**Comment by gianarb** [comment]
@deeky666 fixed! Thanks :D

**Comment by Ocramius** [comment]
Splash!
